### PR TITLE
Create users script

### PR DESCRIPTION
**Pull request by ChaamC**
Script to create or delete a list of users

### CODE REVIEW

File: processUsers.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    users = []
+    for user in user_names:
+        response = session.delete(magpie_url + "/users/" + user)
+        users.append({'user_name': user, 'result': format_response(response)})
+    return users
+
+
+if __name__ == '__main__':
+    email_list = sys.argv[1:]
+    magpie_url = "http://192.168.99.201:2001/magpie"
```
Comment (dbyrns) [inline processUsers.py]: Could use localhost rather than a specific ip here

File: processUsers.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        response = session.delete(magpie_url + "/users/" + user)
+        users.append({'user_name': user, 'result': format_response(response)})
+    return users
+
+
+if __name__ == '__main__':
+    email_list = sys.argv[1:]
+    magpie_url = "http://localhost/magpie"
+    magpie_admin_user_name = "admin"
+    magpie_admin_password = "admin"
```
Comment (fmigneault) [inline processUsers.py]: These parameters should all be passed as input to the script.
Check `argparse` module to make a clean CLI.

File: processUsers.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        output = "USERNAME".ljust(COLUMN_SIZE) + "PASSWORD".ljust(COLUMN_SIZE) + "RESULT".ljust(COLUMN_SIZE) + "\n"
+        output += "".ljust(COLUMN_SIZE * 3, "_") + "\n\n"
+        for user in users:
+            output += user['user_name'].ljust(COLUMN_SIZE) + user['password'].ljust(COLUMN_SIZE) + user['result'].ljust(
+                COLUMN_SIZE) + "\n"
+
+        filename = "createUsers_log__" + datetime.datetime.now().strftime('%Y%m%d__%H%M%S') + ".txt"
+        file = open(filename, "w+")
+        file.write(output)
+        file.close()
```
Comment (fmigneault) [inline processUsers.py]: use a `with(...)` block to auto handle closing file in case of error

File: processUsers.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+        print(output)
+        print("Output results sent to " + filename)
```
Comment (fmigneault) [inline processUsers.py]: Replace all `print` by a logger with `logging` module. (see `magpie.utils` for examples of simple setup of a logger).
Makes it easier to update where/how output info is provided when deployed somewhere using a logging config.

File: processUsers.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
```
Comment (fmigneault) [inline processUsers.py]: rename the file `create_users.py` and move it under `magpie/helpers`

**Pull request merged**